src/images/orcestra-logo.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg id="ORCESTRA" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 558.77 63">
+  <defs>
+    <style>
+      .cls-1 {
+        fill: #00267f;
+      }
```

Comment:
Maybe a bit more elegant and a bit more fun than using javascript to detect the color scheme could be to use the media-query within the SVG itself. This way, the logo adjusts itself to the current color scheme.

```suggestion
      }
      @media (prefers-color-scheme: dark) {
        .cls-1 {
          fill: rgba(255, 255, 255, 0.87);
        }
      }
```

... in theory, it should be possible to refer to the global color variables from the SVG as well. But in that case, the SVG becomes unusable outside the webpage (which might however be fine).